asf_core_data/getters/mcs/get_mcs_installations.py

```diff
@@ -0,0 +1,74 @@
+# File: asf_core_data/getters/mcs/get_mcs_installations.py
+"""Function to get concatenated installations data in a usable format,
+with correct dtypes."""
+
+import pandas as pd
+import os
+
+from asf_core_data import PROJECT_DIR, get_yaml_config
+
+from asf_core_data.getters.data_getters import load_s3_data
+
+config = get_yaml_config(PROJECT_DIR / "asf_core_data/config/base.yaml")
+
+BUCKET_NAME = config["BUCKET_NAME"]
+INSTALLATIONS_RAW_S3_PATH = config["INSTALLATIONS_RAW_S3_PATH"]
+INSTALLATIONS_RAW_LOCAL_PATH = config["INSTALLATIONS_RAW_LOCAL_PATH"]
+RAW_DATA_S3_FOLDER = config["RAW_DATA_S3_FOLDER"]
```

Comment:
For use as a package, we'll have to transfer all the config information into base_config.py since packages don't include any data files - and that's the simplest way to adapt to that. After we're done reviewing, we can go through and change all config references at once. Doesn't have to happen right away, just mentioning it here already. 

It will probably look something like this, only minor adjustments needed.

```suggestion
from config import base_config

BUCKET_NAME = base_config.BUCKET_NAME
INSTALLATIONS_RAW_S3_PATH = base_config.INSTALLATIONS_RAW_S3_PATH
```

Reply:
I've made a new issue for making MCS work as a package and have copied over the relevant comments/suggestions